core/src/main/scala/org/apache/spark/broadcast/TorrentBroadcast.scala

```diff
@@ -54,7 +54,7 @@ import org.apache.spark.util.io.{ChunkedByteBuffer, ChunkedByteBufferOutputStrea
  * @param obj object to broadcast
  * @param id A unique identifier for the broadcast variable.
  */
-private[spark] class TorrentBroadcast[T: ClassTag](obj: T, id: Long)
+private[spark] class TorrentBroadcast[T: ClassTag](@transient val obj: T, id: Long)
```

Comment:
OK. I see. As there is no other places except `val numBlocks: Int = writeBlocks(obj)` use it, there is no field created for it.